docs/configuration-reference.md

```diff
@@ -137,6 +138,37 @@ key | value | description
 :--|:--|:--
 field | string | The name of the field to drop.
 
+### drop_event
+
+The `drop_event` processor will remove all events where the specified field
+exactly matches one of the listed values.
+This can be used to filter datasets from ingress to service based on which
+service or namespace is used.
+
+Events that do not have a key named `field` present will be kept.
```

Comment:
What is the significance of the name "field" here? Are you trying to say that the event has to contain a field with the name nominated in the processor configuration's "field" field?